components/SecondaryButton/SecondaryButton.styled.js

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+
+export const SecondaryButton = styled.button`
+    display: inline-flex;
```

Comment:
Du könntest du auch nur einen Button machen und dann machst du sowas wie `background: props => props.secondary ? var(--me-color-primary-color-white) : var(--me-color-primary-color-gray-700)` und das gleiche auch bei den anderen css properties. https://styled-components.com/docs/basics#adapting-based-on-props. Auch hier finde ich es mit Klassen einfacher, da du einfach sowas wie .button haben kannst und dann zusätzlich .button-primary und .button-secondary.

Reply:
I implemented one button with a props variant for the secondary button. Shall I install `npm install styled-components @types/styled-components` to make sure the styled components work smoothly with Typescript?